DepositeCalcTests/Tests/HistoryPageTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using DepositeCalcTests.Pages;
+using System.Threading;
+
+namespace DepositeCalcTests.Tests
+{
+    internal class HistoryPageTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
+            driver.Url = "https://localhost:5001/History";
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+
+        [Test]
+        public void CalculatorLinkTest()
+        {
+            // Arrange
+            var historyPage = new HistoryPage(driver);
+
+            // Act
+            var calculatorPage = historyPage.Calculator();
+
+            // Assert
+            Assert.IsTrue(calculatorPage.IsOpened(), "Incorrect page");
+        }
+
+        [Test]
+        public void TitleTest()
+        {
+            // Arrange
+            string historyPageTitle = "History";
+
+            // Assert
+            Assert.AreEqual(historyPageTitle, driver.Title, "Incorrect title");
```

Comment:
It could be done all in one line:
```suggestion
            Assert.AreEqual("History", driver.Title, "Incorrect title");
```